puppet/files/treeherder/.bash_aliases

```diff
@@ -90,3 +91,11 @@ function thresetall {
     echo "Purging queues"
     thqueuespurge
 }
+
+function thpurge {
+    ~/treeherder/manage.py init_datasources --reset
+    ~/treeherder/manage.py flush
+    thresetall
+    echo
+    echo "Please exit vagrant and run 'vagrant provision' to finish"
```

Comment:
They can run vagrant provision in another shell, they don't need to exit, perhaps:
"Please run 'vagrant provision' from the host to finish"